exercises/practice/grade-school/GradeSchoolTest.php

```diff
@@ -41,26 +19,96 @@ protected function setUp(): void
         $this->school = new School();
     }
 
-    public function testAddStudent(): void
+    /**
+     * uuid: 9337267f-7793-4b90-9b4a-8e3978408824
+     * @testdox Add a student
+     */
+    public function testAddAStudent(): void
+    {
+        $this->school->add('Aimee', 2);
+        $this->assertCount(1, $this->school->grade(2));
+    }
+
+    /**
+     * uuid: 6d0a30e4-1b4e-472e-8e20-c41702125667
+     * @testdox Student is added to the roster
+     */
+    public function testStudentIsAddedToTheRoster(): void
     {
-        $this->school->add('Claire', 2);
-        $this->assertContains('Claire', $this->school->grade(2));
+        $this->school->add('Aimee', 2);
+        $this->assertContains('Aimee', $this->school->grade(2));
     }
 
-    public function testAddStudentsInSameGrade(): void
+    /**
+     * uuid: 73c3ca75-0c16-40d7-82f5-ed8fe17a8e4a
+     * @testdox Adding multiple students in the same grade in the roster
+     */
+    public function testAddingMultipleStudentsInTheSameGradeInTheRoster(): void
     {
-        $this->school->add('Marc', 2);
-        $this->school->add('Virginie', 2);
-        $this->school->add('Claire', 2);
+        $this->school->add('Blair', 2);
+        $this->school->add('James', 2);
+        $this->school->add('Paul', 2);
 
         $students = $this->school->grade(2);
         $this->assertCount(3, $students);
         $this->assertEqualsCanonicalizing(
-            ['Claire', 'Marc', 'Virginie'],
+            ['Blair', 'James', 'Paul'],
             array_values($students)
         );
     }
 
+    /**
+     * uuid: 233be705-dd58-4968-889d-fb3c7954c9cc
+     * @testdox Multiple students in the same grade are added to the roster
+     */
+    public function testMultipleStudentsInTheSameGradeAreAddedToTheRoster(): void
+    {
+        $this->school->add('Blair', 2);
+        $this->school->add('James', 2);
+        $this->school->add('Paul', 2);
+
+        $students = $this->school->grade(2);
+        $this->assertCount(3, $students);
+        $this->assertEquals(
```

Comment:
To allow any solution with an array (associative or not, sorted or not), we need to use `assertEqualsCanonicalizing()`:

```suggestion
        $this->assertEqualsCanonicalizing(
```